extensions/wikia/CreateNewWiki/js/WikiWelcome.js

```diff
@@ -3,10 +3,17 @@
 /**
  * A module that displays a "welcome" modal when the CreateNewWiki process is finished
  */
-define('WikiWelcome', ['wikia.ui.factory', 'wikia.loader'], function (uiFactory, loader) {
+define('WikiWelcome', ['wikia.ui.factory', 'wikia.loader', 'wikia.tracker'], function (uiFactory, loader, tracker) {
 	'use strict';
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AStrictMode)